napari/layers/utils/interactivity_utils.py

```diff
@@ -51,7 +52,7 @@ def displayed_plane_from_nd_line_segment(
 
 def drag_data_to_projected_distance(
     start_position, end_position, view_direction, vector
```

Comment:
Should these arguments get types too, e.g `np.ndarray`?

Reply:
Probably, but the intent here was to type `layers.base` and bits in other files that were required for that added typing, so I'd rather leave this for now.

Reply:
ok, and just so I further understand: `utils.interactivity_utils.py` isn't checked by mypy so the partial typing doesn't cause any squeals, right?